cli/crates/dev-server/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "grafbase-local-dev-server"
```

Comment:
What if we used the name `grafbase-dev-server`. `dev-server` implies it's local IMO.

Reply:
I was keeping the same convention, we could go with either that or `grafbase-local-server`

Reply:
`grafbase-local-server` works

Reply:
Done